MovieCave - MVVM/MovieCave/MovieCave/Custom UI/PopUp/PopUpView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  PopUpView.swift
+//  MovieCave
+//
+//  Created by Admin on 27.10.23.
+//
+
+import UIKit
+
+class PopUpView: UIView {
+
+    //MARK: - IBOutlets
+    @IBOutlet private weak var messageLabel: UILabel!
+    
+    //MARK: - Properties
+    private var autoDismissTimer: Timer?
+    
+    //MARK: - Intializers
+    required init?(coder: NSCoder) {
```

Comment:
Why do you need this init?

Reply:
"'required' initializer 'init(coder:)' must be provided by subclass of 'UIView'"